File: app/src/main/java/com/example/pico_botella/model/Reto.java
package com.example.pico_botella.model;

public class Reto {
    private int id;
    private String descripcion;

    public Reto(int id, String descripcion) {
        this.id = id;
        this.descripcion = descripcion;
    }

    public Reto(String descripcion) {
        this.descripcion = descripcion;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }
}

File: app/src/main/java/com/example/pico_botella/model/Reto.kt
package com.example.pico_botella.model

import com.google.firebase.firestore.Exclude
import com.google.firebase.firestore.IgnoreExtraProperties
import com.google.firebase.firestore.ServerTimestamp
import java.util.Date

@IgnoreExtraProperties
data class Reto(
    @get:Exclude var id: String = "",
    var descripcion: String = "",
    @ServerTimestamp var createdAt: Date? = null
)
